### DISCUSSION TITLE
Compiler Explorer: format chrono returns 3221226505

### DISCUSSION THREAD

**Discussion by H-G-Hristov** [microsoft/STL#3329] [Q&A] [answered]
https://godbolt.org/z/vKd3717To

Could that be an implementation bug?

![Screenshot 2023-01-07 at 13 19 14](https://user-images.githubusercontent.com/47526411/211147548-f8898c08-2f90-4b01-a824-67d7fb9c55d9.png)

**Comment by fsb4000** [comment]
That might be godbolt limitations or docker limitations.

Your example works for me:

![изображение](https://user-images.githubusercontent.com/4289847/211191682-4691d16d-00d6-4789-b828-6b77cb52d3f9.png)

**Comment by cpplearner** [comment, marked as answer]
The OS used by Compiler Explorer is a bit too old to have the data needed by the current implementation of `utc_clock`.

https://github.com/microsoft/STL/wiki/VS-2019-Changelog#vs-2019-1610:
> While the STL generally provides all features on all supported versions of Windows, leap seconds and time zones (which change over time) require OS support that was added to Windows 10. Specifically, updating the leap second database requires Windows 10 version 1809 or later, and time zones require icu.dll which is provided by Windows 10 version 1903/19H1 or later. This applies to both client and server OSes; note that Windows Server 2019 is based on Windows 10 version 1809.

See also https://github.com/compiler-explorer/compiler-explorer/issues/3500
